.github/workflows/test-compilation.yml

```diff
@@ -14,44 +15,38 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test-crate-compilation:
-    name: '${{ matrix.crate }} on ${{ matrix.os }}'
```

Comment:
I'd leave/adjust this explicit name here to shorten the CI name from e.g.`compilation (ubuntu-latest / schema-engine-cli)` to `compile schema-engine-cli @ ubuntu-latest`.